src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -81,4 +83,9 @@ def get(self, request: Request, project) -> Response:
 
         context = {"actions": action_list, "conditions": condition_list, "filters": filter_list}
 
+        if request.GET.get("includeAllTickets") is not None:
```

Comment:
Very open to alternative names for the query parameter `includeAllTickets` or the response key `disabledTicketActions`

Reply:
does it need to take into account `can_create_tickets`

Reply:
ah good catch, added that in